### PR TITLE
add zenodo authors file

### PR DESCRIPTION
**Pull request by fgrunewald**
In order to push martinize2 to Zenodo and have the authors being detected automatically.

### CODE REVIEW

**Review by pckroon — COMMENTED** [review]
@jbarnoud @Tsjerk Do you guys have ORCIDs?

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        {
+            "orcid": "0000-0001-9273-4850",
+            "name": "Peter C Kroon"
+        },
+        {
+            "orcid": "0000-0001-6979-1363",
+            "name": "Fabian Gr\"unewald"
+        },
+        {
+            "orcid": "NaN",
```
Comment (jbarnoud) [inline .zenodo.json:12]:
```suggestion
            "orcid": "0000-0003-0343-7796",
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            "orcid": "0000-0003-0343-7796",
+            "name": "Jonathan Barnoud"
+        },
+        {
+            "orcid": "0000-0003-0660-1301",
+            "name": "Paulo C. T. Souza"
+        },
+        {
+            "orcid": "NaN",
+            "name": "Tsjerk A Wassenaar"
```
Comment (pckroon) [inline .zenodo.json:21]:
```suggestion
            "orcid": "0000-0002-6345-0266",
            "name": "Tsjerk A Wassenaar"
```

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "creators": [
+        {
+            "orcid": "0000-0001-9273-4850",
+            "name": "Peter C Kroon"
```
Comment (pckroon) [inline .zenodo.json:5]:
```suggestion
            "name": "Peter C. Kroon"
```

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            "orcid": "0000-0003-0343-7796",
+            "name": "Jonathan Barnoud"
+        },
+        {
+            "orcid": "0000-0003-0660-1301",
+            "name": "Paulo C. T. Souza"
+        },
+        {
+            "orcid": "0000-0002-6345-0266",
+            "name": "Tsjerk A Wassenaar"
```
Comment (pckroon) [inline .zenodo.json:21]:
```suggestion
            "name": "Tsjerk A. Wassenaar"
```

**Review by pckroon — APPROVED** [review]
Going full nitpick mode: let's do the names consistently.

**Review by pckroon — APPROVED** [review]

**Pull request merged**